extensions.json

```
@@ -852,8 +850,6 @@
     {
       "id": "justusadam.language-haskell",
       "repository": "https://github.com/JustusAdam/language-haskell",
```

`3.4.0` is still the latest official version.

```suggestion
      "repository": "https://github.com/JustusAdam/language-haskell",
      "version": "3.4.0"
      "checkout": "v3.4.0",
```